configure

```diff
@@ -17,7 +17,13 @@
 
 # Generate $LINKpackage variable
 genlink() {
-  eval "LINK$1=\"\\\$(EXTLIB)/lib/lib$1$2\\\$(LIBSUFFIX)\""
+  if [[ "${IGNORED_LIBS[@]}" =~ "$1" ]]; then
```

Comment:
Why is this needed?

Reply:
Discord was messing with android builds so I added an ignore array

It is no longer used anymore so it may be removed